prow/plugins/buildifier/buildifier.go

```diff
@@ -125,8 +125,8 @@ func uniqProblems(problems []string) []string {
 
 // problemsInFiles runs buildifier on the files. It returns a map from the file to
 // a list of problems with that file.
-func problemsInFiles(r git.RepoClient, files map[string]string) (map[string][]string, error) {
-	problems := make(map[string][]string)
+func problemsInFiles(r git.RepoClient, files map[string]string) (map[string]bool, error) {
+	problems := map[string]bool{}
```

Comment:
Not quite sure why this is also part of the PR, is it intended?

Reply:
Yes, `build.Rewrite()` no longer exists at the updated version of github.com/bazelbuild/buildtools/build that repo-infra uses.